packages/radix-ui-themes/src/components/text-area.css

```diff
@@ -45,37 +57,50 @@
   }
 }
 
+.rt-TextAreaChrome {
```

Comment:
So this now uses exactly the same technique as `TextField`?

Reply:
Yeah, can't have nice blending of the border with the background because to do that, we either need:
- `padding: 1px; background-clip: padding-box;` on the element that has the shadow
- or, a transparent border and another absolutely positioned element for the shadow

We already use padding on the text area, and a pseudo is not an option on this tag.

What we had before with the transparent border was a broken mix of both techniques—not sure how I came up with that 😵‍💫